utils/check_repo.py

```diff
@@ -55,6 +55,7 @@
     "DPRSpanPredictor",  # Building part of bigger (documented) model.
     "T5Stack",  # Building part of bigger (tested) model.
     "TFElectraMainLayer",  # Building part of bigger (documented) model (should it be a TFPreTrainedModel ?)
+    "BlenderbotForConditionalGeneration",
```

Comment:
You're going to anger @sgugger :)

Reply:
This should not be there indeed. The suggestions above should remove the need for it.